addon/mixins/scoped-query-cache.js

```diff
@@ -30,29 +42,19 @@ export default Ember.Mixin.create({
           formatCacheKey(query) :
           query;
 
-    assert('queryCache needs to be an object', Ember.typeOf(queryCache) === 'object');
+    assert('queryCache needs to be an object', typeOf(queryCache) === 'object');
     return JSON.stringify(sortedObject(queryCache));
   },
 
-  queryCache(modelName, query, options = {}, queryMethod) {
+  getFromCache(modelName, query, options = {}, queryMethod) {
```

Comment:
This mixin is expanding the interface of the object it is being mixed into quite substantially. I thought the design was to move everything to a namespace? The exception being the namespace object itself `queryCache` and `init` `destroy` super hooks.

* `this.queryCache.get`
* `this.queryCache.find`
* `this.queryCache.getStringifiedQuery`
* etc.